easy_gui/master_classes.py

```diff
@@ -682,6 +685,21 @@ def flush(self):
         pass
 
 
+class Progressbar(easy_gui.Widget):
+    def __init__(self, master=None, length = 100, mode = 'determinate', **kwargs) -> None:
+        super().__init__(master=master, **kwargs)
+        self.length = length
+        del kwargs['grid_area']
```

Comment:
I don't know if this deletion is necessary, i just copied it from the Label class. Maybe it is for some reason, I'll test it tomorrow.

Reply:
The "grid_area" kwarg is used for CSS-layout positioning if specified.  The easy_gui.Widget class uses it (passed to super() in line 690), but the base tkinter/ttk classes get upset when it is passed to them (like in 693 here)--hence the deletion if it exists in the kwargs dict.